integration-tests/contract-transfer-ticket-between-implicit-and-originated-accounts.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { CONFIGS } from "./config";
+import { Protocols, DefaultContractType, TezosToolkit } from "@taquito/taquito";
+import { ticketsSendTz, ticketsBagTz, ticketsBlackholeTz } from "./data/code_with_ticket_transfer";
+import { RpcClient, TicketTokenParams } from '@taquito/rpc';
+
+CONFIGS().forEach(({ lib, protocol, rpc, setup, createAddress }) => {
+  const Tezos1 = lib;
+  const client = new RpcClient(rpc);
+  const mumbaiAndAlpha = protocol === Protocols.PtMumbaii || protocol === Protocols.ProtoALpha ? test : test.skip;
+  let tezos1Pkh: string;
+  let Tezos2: TezosToolkit;
+  let ticketSendContract: DefaultContractType;
+  let ticketBagContract: DefaultContractType;
+  let ticketBlackholeContract: DefaultContractType;
+  let ticketToken: TicketTokenParams
+
+  describe(`Test contract origination of a simple contract through contract api using: ${rpc}`, () => {
+
+    beforeAll(async (done) => {
```

Comment:
some indents in this test are 4 spaces and 2 spaces. could you make them all 2 spaces?